libnmstate/nm/ovs.py

```diff
@@ -57,19 +62,34 @@ def create_bridge_setting(options_state):
 
 def create_port_setting(port_state):
     port_setting = nmclient.NM.SettingOvsPort.new()
+
+    lag_state = port_state.get(OB.Port.LINK_AGGREGATION_SUBTREE)
+    if lag_state:
+        mode = lag_state.get(OB.Port.LinkAggregation.MODE)
+        if mode == OB.Port.LinkAggregation.Mode.LACP:
```

Comment:
It seems to me that tests for the different modes are missing/conditionals are missing.

Reply:
Done